code/services/ExtensibleSearchService.php

```diff
@@ -155,4 +155,37 @@ public function getSuggestions($term, $pageID, $limit = 5, $approved = true) {
 		return null;
 	}
 
+
+	/**
+	 *	Retrieve the most relevant search suggestions.
+	 *
+	 *	@parameter <{SEARCH_TERM}> string
+	 *	@parameter <{EXTENSIBLE_SEARCH_PAGE_ID}> integer
+	 *	@return array
+	 */
+
+	public function getTypeahead($request, $pageID) {
```

Comment:
Wasn't this going to be changed to just take the searchterm directly as a param instead of the whole $request object? Now that you're not using it as a request obj...